Sockets/Sockets.Implementation.NET/TcpSocketClient.cs

```diff
@@ -36,11 +38,41 @@ internal TcpSocketClient(TcpClient backingClient)
         /// </summary>
         /// <param name="address">The address of the endpoint to connect to.</param>
         /// <param name="port">The port of the endpoint to connect to.</param>
-        public async Task ConnectAsync(string address, int port)
+        /// <param name="secure">Is this connection secure?</param>
+        public async Task ConnectAsync(string address, int port, bool secure = false)
         {
             await _backingTcpClient.ConnectAsync(address, port);
+            if (!secure)
```

Comment:
Yes you are right. My bad. It was !secure, I did a clean up before push and forgot to change that. Can you change at your end or should a do here and push again? (sorry, I'm a bit noob on git, used to work with TFS)